x-pack/plugin/fleet/src/main/java/org/elasticsearch/xpack/fleet/Fleet.java

```diff
@@ -186,4 +200,25 @@ private SystemIndexDescriptor fleetArtifactsSystemIndexDescriptors() {
     private String loadTemplateSource(String resource) {
         return TemplateUtils.loadTemplate(resource, Version.CURRENT.toString(), MAPPING_VERSION_VARIABLE);
     }
+
+    @Override
+    public List<ActionHandler<? extends ActionRequest, ? extends ActionResponse>> getActions() {
+        return Arrays.asList(
+            new ActionHandler<>(GetGlobalCheckpointsAction.INSTANCE, GetGlobalCheckpointsAction.TransportGetGlobalCheckpointsAction.class),
```

Comment:
nit: just for consistency we should rename this to just `TransportAction` too? Completely optional.